src/test/java/com/sparta/popupstore/domain/popupstore/service/PopupstoreServiceTest.java

```diff
@@ -0,0 +1,158 @@
+import com.sparta.popupstore.domain.kakaoaddress.service.KakaoAddressService;
+import com.sparta.popupstore.domain.popupstore.bundle.dto.request.PopupStoreImageRequestDto;
+import com.sparta.popupstore.domain.popupstore.bundle.entity.PopupStoreBundle;
+import com.sparta.popupstore.domain.popupstore.bundle.service.PopupStoreBundleService;
+import com.sparta.popupstore.domain.popupstore.dto.request.PopupStoreCreateRequestDto;
+import com.sparta.popupstore.domain.popupstore.dto.response.PopupStoreCreateResponseDto;
+import com.sparta.popupstore.domain.popupstore.entity.PopupStore;
+import com.sparta.popupstore.domain.popupstore.repository.PopupStoreRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+class PopupstoreServiceTest {
+
+    @Mock
+    private PopupStoreRepository popupStoreRepository;
+
+    @Mock
+    private KakaoAddressService kakaoAddressService;
+
+    @Mock
+    private PopupStoreBundleService popupStoreBundleService;
+
+    @InjectMocks
+    private PopupStoreService popupStoreService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
+    @DisplayName("팝업스토어 생성 - 성공")
+    void createPopupStore_success() {
+        // Given
+        Company company = createTestCompany();
+        PopupStoreCreateRequestDto requestDto = createValidRequestDto();
+
+        Address address = new Address(new RoadAddress("Seoul", "Seoul", 37.5665, 126.9780));
+        when(kakaoAddressService.getKakaoAddress(any(String.class))).thenReturn(address);
+
+        PopupStore popupStore = createTestPopupStore(company, address);
+        when(popupStoreRepository.save(any(PopupStore.class))).thenReturn(popupStore);
+
+        PopupStoreBundle bundle = new PopupStoreBundle(
+                List.of(),
+                List.of(),
+                List.of()
+        );
+
+        when(popupStoreBundleService.createPopupStoreBundle(any(), any(), any(), any())).thenReturn(bundle);
+
+        // When
+        PopupStoreCreateResponseDto response = popupStoreService.createPopupStore(company, requestDto);
+
+        // Then
+        verify(kakaoAddressService, times(1)).getKakaoAddress(any(String.class));
+        verify(popupStoreRepository, times(1)).save(any(PopupStore.class));
+        verify(popupStoreBundleService, times(1)).createPopupStoreBundle(any(), any(), any(), any());
+
+        assertNotNull(response);
+    }
+
+    @Test
+    @DisplayName("팝업스토어 생성 - 주소 조회 실패")
+    void createPopupStore_addressLookupFail() {
```

Comment:
```suggestion
    void test1() {
```
DisplayName으로 이름을 정해서 굳이 정확한 이름을 넣지 않아도 되기는 합니다.

Reply:
수정했습니다